backend/src/main/kotlin/xyz/elitese/ehrenamtskarte/importer/general/ImportToDatabase.kt

```diff
@@ -29,25 +28,36 @@ object ImportToDatabase {
         ).replace("<br/>", "\n")
     }
 
+    /**
+     * Replaces notes for unspecified entries with empty strings.
+     *
+     * Such a note consists of the letters "nA" (in that order), potentially with dots like "n.A." and/or stuffed with whitespace (e.g. " n A. ").
+     *
+     * @return empty string if `text` is a "unspecified note", `text` otherwise
+     */
+    private fun replaceUnspecifiedNote(text: String): String {
```

Comment:
The issue says that the entry should be null. In this PR an empty string is used. I think null would be way better

Reply:
You are right. I used the empty string because this is what undefined contact data are currently stored like. Would make more sense to change these entries to null, too 👍

Reply:
Done